_component_design/time-picker.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Time Picker
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: The time picker enables users to select a specific time or window from a sequential list. The time picker is often accompanied by a date selector for the purpose of creating a date range filter.
+preview-image: preview-images/time-picker.svg
+resource: true
+status: stable
+last-modified: 2017-10-24
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+## Introduction
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### When to use
+
+Use the time picker if the list of times are sequential, such as 9 a.m., 10 a.m., 11 a.m., and so on.
```

Comment:
Use AM and PM for all a.m. and p.m. occurrences (per Style Guide)

Reply:
Done.